exercises/practice/wordy/cases_test.go

```diff
@@ -1,159 +1,152 @@
-		"What is 1 plus 5 minus -2?",
-		true,
-		8,
+		description: "addition and subtraction",
+		question:    "What is 1 plus 5 minus -2?",
+		expectError: false,
+		expected:    8,
 	},
 	{
-		"multiple subtraction",
-		"What is 20 minus 4 minus 13?",
-		true,
-		3,
+		description: "multiple subtraction",
+		question:    "What is 20 minus 4 minus 13?",
+		expectError: false,
+		expected:    3,
 	},
 	{
-		"subtraction then addition",
-		"What is 17 minus 6 plus 3?",
-		true,
-		14,
+		description: "subtraction then addition",
+		question:    "What is 17 minus 6 plus 3?",
+		expectError: false,
+		expected:    14,
 	},
 	{
-		"multiple multiplication",
-		"What is 2 multiplied by -2 multiplied by 3?",
-		true,
-		-12,
+		description: "multiple multiplication",
+		question:    "What is 2 multiplied by -2 multiplied by 3?",
+		expectError: false,
+		expected:    -12,
 	},
 	{
-		"addition and multiplication",
-		"What is -3 plus 7 multiplied by -2?",
-		true,
-		-8,
+		description: "addition and multiplication",
+		question:    "What is -3 plus 7 multiplied by -2?",
+		expectError: false,
+		expected:    -8,
 	},
 	{
-		"multiple division",
-		"What is -12 divided by 2 divided by -3?",
-		true,
-		2,
+		description: "multiple division",
+		question:    "What is -12 divided by 2 divided by -3?",
+		expectError: false,
+		expected:    2,
 	},
 	{
-		"unknown operation",
-		"What is 52 cubed?",
-		false,
-		0,
+		description: "unknown operation",
+		question:    "What is 52 cubed?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"Non math question",
-		"Who is the President of the United States?",
-		false,
-		0,
+		description: "Non math question",
+		question:    "Who is the President of the United States?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"reject problem missing an operand",
-		"What is 1 plus?",
-		false,
-		0,
+		description: "reject problem missing an operand",
+		question:    "What is 1 plus?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"reject problem with no operands or operators",
-		"What is?",
-		false,
-		0,
+		description: "reject problem with no operands or operators",
+		question:    "What is?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"reject two operations in a row",
-		"What is 1 plus plus 2?",
-		false,
-		0,
+		description: "reject two operations in a row",
+		question:    "What is 1 plus plus 2?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"reject two numbers in a row",
-		"What is 1 plus 2 1?",
-		false,
-		0,
+		description: "reject two numbers in a row",
+		question:    "What is 1 plus 2 1?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"reject postfix notation",
-		"What is 1 2 plus?",
-		false,
-		0,
+		description: "reject postfix notation",
+		question:    "What is 1 2 plus?",
+		expectError: true,
+		expected:    0,
 	},
 	{
-		"reject prefix notation",
-		"What is plus 1 2?",
-		false,
-		0,
-	},
-	{
-		"reject using default 0 minus number",
```

Comment:
It seems this test is no longer present, but the `tests.toml` was not updated?

Reply:
Seems like it was added directly to `cases_test.go`: https://github.com/exercism/go/commit/e1a2b21f85a930e42bd683e8073e233fa6c16a2b

Reply:
Should I add it to `wordy_test.go` and then mention it in #1892 as additional test?

Reply:
Let's not include this test then. I'll make a note to investigate this further later, but at first sight it seems that if this test is important, it should have been added directly to the problem specifications repo. It doesn't seem a test related to any go-specific change to the exercise.